src/_gettsim/arbeitslosengeld_2/regelbedarf.py

```diff
@@ -96,47 +225,29 @@ def mehrbedarf_alleinerziehend_m(
 def kindersatz_m_bis_2010(
     alter: int,
     kindergeld__gleiche_fg_wie_empfänger: bool,
-    arbeitsl_geld_2_params: dict,
+    regelsatz_anteilsbasiert: RegelsatzAnteilsbasiert,
 ) -> float:
-    """Basic monthly subsistence / SGB II needs of children until 2010.
-
-    Parameters
-    ----------
-    alter
-        See basic input variable :ref:`alter`.
-    kindergeld__gleiche_fg_wie_empfänger
-        See :func:`kindergeld__gleiche_fg_wie_empfänger`.
-    arbeitsl_geld_2_params
-        See params documentation :ref:`arbeitsl_geld_2_params <arbeitsl_geld_2_params>`.
-
-    Returns
-    -------
-    float with SGB II needs of children until year 2010.
-
-    """
-    basissatz = arbeitsl_geld_2_params["regelsatz_anteilsbasiert"]["basissatz"]
-    anteile = arbeitsl_geld_2_params["regelsatz_anteilsbasiert"][
-        "anteil_vom_basissatz_für_kinder"
-    ]
+    """Basic monthly subsistence / SGB II needs of children until 2010."""
+    basissatz = regelsatz_anteilsbasiert.basissatz
 
     if (
-        alter >= anteile["zwischen_14_und_24_jahre"]["min_alter"]
-        and alter <= anteile["zwischen_14_und_24_jahre"]["max_alter"]
+        alter >= regelsatz_anteilsbasiert.kind.zwischen_14_und_24_jahre.min_alter
```

Comment:
Is there a specific term used in the law? It really is ugly to have the age appearing twice here.